qa/tasks/notification_tests.py

```diff
@@ -12,47 +12,28 @@
 
 from teuthology import misc as teuthology
 from teuthology import contextutil
-from teuthology.config import config as teuth_config
 from teuthology.orchestra import run
 
 log = logging.getLogger(__name__)
 
+
 @contextlib.contextmanager
 def download(ctx, config):
-    """
-    Download the bucket notification tests from the git builder.
-    Remove downloaded test file upon exit.
-    The context passed in should be identical to the context
-    passed in to the main task.
-    """
     assert isinstance(config, dict)
-    log.info('Downloading bucket-notification-tests...')
+    log.info('Downloading bucket-notifications-tests...')
     testdir = teuthology.get_testdir(ctx)
+    branch = ctx.config.get('suite_branch', None)
```

Comment:
what if you don't explicitly provide these values?
does it work in teuthology without them by taking the defaults?

Reply:
I was about to ask this question but I think I've a way to solve this which I'll try it out.